libs/SalesforceSDK/res/values/strings.xml

```diff
@@ -12,5 +12,5 @@
    	<string name="oauth_display_type">touch</string>
 
    	<!-- Default API version used by the SDK and sample apps -->
-   	<string name="api_version">v36.0</string>
```

Comment:
38 is required for composite - since 39 is out, we might as well use it by default